modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -15,9 +15,58 @@ The following steps show how to integrate Service Mesh with user-workload monito
 
 * User-workload monitoring is enabled.
 * {SMProductName} Operator 2.4 is installed.
+* Kiali Operator 1.65 is installed.
 
 .Procedure
 
+. Create a token to Thanos for Kiali:
++
+[source,terminal]
+----
+$ SECRET=`oc get secret -n openshift-user-workload-monitoring | 
+ grep  prometheus-user-workload-token | head -n 1 | awk '{print $1 }'`
+----
++
+[source,terminal]
+----
+$ TOKEN=`echo $(oc get secret $SECRET -n openshift-user-workload-monitoring -o jsonpath='{.data.token}' | base64 -d`
```

Comment:
```suggestion
$ TOKEN=`echo $(oc get secret $SECRET -n openshift-user-workload-monitoring -o jsonpath='{.data.token}') | base64 -d`
```
It would be great if we could remove `$`, because it does not allow to copy the line and paste in the terminal as is. Instead, we have to copy and remove first character. This is so annoying... Is it necessary?

Reply:
Yes, unfortunately, the `$` has to remain per Red Hat style guidelines:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

If the user must run a command as root, use a number sign (#) at the start of the command instead of a dollar sign ($). For example:

# subscription-manager list

Reply:
@jewertow and @gwynnemonahan - FYI, when a user clicks the Copy icon next to sample code in the docs, the prompt is not included in the copied text:

![image](https://github.com/openshift/openshift-docs/assets/86380613/6d3e1967-a1bf-4c85-ba3b-a9a85bf78f9b)